src/components/tabController/index.tsx

```diff
@@ -204,4 +237,8 @@ TabController.TabBar = TabBar;
 TabController.TabBarItem = TabBarItem;
 TabController.TabPage = TabPage;
 TabController.PageCarousel = PageCarousel;
-export default asBaseComponent(TabController);
+export default asBaseComponent<
+  TabControllerProps,
+  typeof TabController
+  //@ts-ignore typescript - will be fixed when moved to functional component
+>(TabController);
```

Comment:
no need for asBaseComponent

Reply:
It seems it was `asBaseComponent` to begin with, I see we're using its `config` in private for `carouselPageWidth`.

Reply:
you are completely right, I accidentally looked at the old TabController file that sits under Incubator 🤦‍♂️

Reply:
Yes, tell me about it, I spent some time trying to figure about why my changes are not applied in VSCode (why are my files still in javascript)... :)

Reply:
yes, we plan to remove these files in V6. 
My bad, I should have warn you about it..

Reply:
No worries :)